_layouts/base.html

```diff
@@ -0,0 +1,158 @@
+<!doctype html>
+<html lang="en">
+
+  <head>
+    <!-- Google Tag Manager --
+    <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':
+    new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],
+    j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src=
+    'https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);
+    })(window,document,'script','dataLayer','{{ tag_manager_code }}');</script>
+    !-- End Google Tag Manager -->
+
+    <title>Webteam practices</title>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="icon" href="https://assets.ubuntu.com/v1/84489620-picto-knowledge-orange.svg" type="image/x-icon" />
+    <link rel="stylesheet" href="/practices/css/styles.css" />
+  </head>
+
+  <body>
+    <!-- Google Tag Manager (noscript) --
+    <noscript><iframe src="https://www.googletagmanager.com/ns.html?id={{ tag_manager_code }}"
+    height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
+    !-- End Google Tag Manager (noscript) -->
+
+    <div class="docs-container">
+      <header id="navigation" class="p-navigation">
+        <div class="p-navigation__banner">
+          <div class="p-navigation__logo">
+            <a class="p-navigation__link" href="httos://canonical-webteam.github.io/practices">
```

Comment:
You've invented the "httos" protocol here :P

Reply:
That's not a protocol, it's the HyperText Transfer Obelisk (Secure)